frontend/services/gui/GUI_Service.cpp

```diff
@@ -61,7 +60,8 @@ bool GUI_Service::init(const Config& config) {
         "ProjectLoader",                         // 8 - trigger loading of new running project
         "FrameStatistics",                       // 9 - current fps and ms value
         "RuntimeConfig",                         // 10 - resource paths
-        "WindowManipulation"                     // 11 - GLFW window pointer
+        "WindowManipulation",                    // 11 - GLFW window pointer
+        "ScreenshotGUIRenderRequest"             // 12 - gui render request from screenshot service
```

Comment:
It would be easier if the GUI Service does not need to know when other Services want to add a GUI window. For example the GUI Service could provide a "register GUI window callback" resource which other services can use to register ImGUI callbacks themselves.

Reply:
see commit b3fe3c4